### ISSUE TITLE
faker seems to be broken

### ISSUE THREAD

**Issue by banhill** [unmock/unmock-js#272]
Examples I get for `u.string("name.firstName")`:
- 'tempor exercitation'
- 'non fugiat Lorem id'
- 'ex tempor consequat minim'
- 'Ut ullamco minim magna aliquip'
- ...

versions:
```
  "devDependencies": {
    "@types/jest": "^24.0.18",
    "jest": "^24.9.0",
    "prettier": "^1.18.2",
    "ts-jest": "^24.1.0",
    "typescript": "^3.6.3",
    "unmock": "^0.3.4"
  },
```

**Comment by idantene** [comment]
Hey @banhill! Thanks, that's definitely an oversight.
It should be fixed after we merge #289, and we'll have a new version out today.

**Comment by idantene** [comment]
#289 is merged into `dev` and a new version will be out soon. Thanks!
